components/infobox/wikis/starcraft2/infobox_team_custom.lua

```diff
@@ -74,19 +76,9 @@ end
 
 function CustomInjector:parse(id, widgets)
 	if id == 'earnings' then
-		CustomTeam.calculateEarnings(_args)
-		local earningsDisplay
-		if _team.totalEarnings > 0 then
-			earningsDisplay = '$' .. _LANGUAGE:formatNum(_team.totalEarnings)
-		end
-		local earningsFromPlayersDisplay
-		if _team.totalEarningsWhileOnTeam > 0 then
-			earningsFromPlayersDisplay = '$' .. _LANGUAGE:formatNum(_team.totalEarningsWhileOnTeam)
-		end
-		return {
-			Cell{name = 'Approx. Total Winnings', content = {earningsDisplay}},
-			Cell{name = _PLAYER_EARNINGS_ABBREVIATION, content = {earningsFromPlayersDisplay}},
-		}
+		table.insert(widgets, Cell{name = _PLAYER_EARNINGS_ABBREVIATION,
+			content = {_team.playerEarnings and '$' .. _LANGUAGE:formatNum(_team.playerEarnings) or nil}
+		})
```

Comment:
this is a change in display

Reply:
First one will have Abbr is the only change right?

Reply:
yeah